packages/devtools_app/lib/src/standalone_ui/api/vs_code_api.dart

```diff
@@ -71,6 +77,12 @@ abstract interface class VsCodeApi {
   static const openDevToolsPageDebugSessionIdParameter = 'debugSessionId';
   static const openDevToolsPagePageParameter = 'page';
 
+  static const hotReloadMethod = 'hotReload';
+  static const hotReloadDebugSessionIdParameter = 'debugSessionId';
+
+  static const hotRestartMethod = 'hotRestart';
+  static const hotRestartDebugSessionIdParameter = 'debugSessionId';
```

Comment:
maybe we should have a single parameter `debugSessionIdParameter` that can be reused instead of duplicating this three times.

Reply:
Done! I also renamed the others so they're just the strings (with `json` prefix - something I'd been inconsistent with - and a suffix for method/event/parameter). I'd like to code-gen this code really (because copy/pasting it and adding new methods seems error-prone) but it hasn't yet felt worth the effort.